benchmarks/src/environment/snake.py

```diff
@@ -0,0 +1,64 @@
+from typing import Dict, Any
+
+import dataclasses
+import logging
+import os
+
+from pathlib import Path
+# from snakemake import snakemake
```

Comment:
The user that creates the benchmark should not be forced to have `snakemake` installed. Therefore snakemake should only be imported when the benchmark is actually being executed. I suggest to import it locally in the `submit` method.